### PR TITLE
add content to about.rst

### PR DESCRIPTION
**Pull request by bwang64**

### CODE REVIEW

File: docs/about.rst [hunk 1]
```diff
@@ -3,4 +3,16 @@ Why `repytah`
...
+to each other in order to form aligned hierarchies, a low-dimensional representation for sequential data
+that synthesizes and aligns all possible hierarchies of the meaningful repetitive structure in a song along
+a common time axis. In addition to visualizing the structure of a song, aligned hierarchies can be
+embedded into a classification space with a natural notion of distance and post-processed to narrow the
+exploration of a music-based data stream to certain lengths of structure, or to address numerous MIR
+tasks, including the cover song task, the segmentation task, and the chorus detection task.
+
+For future work, based on the aligned hierarchies, we can build aligned sub-hierarchies. Aligned sub-
+hierarchies is a collection of individual aligned hierarchies for each repeated pattern in a given sequential
+data. With aligned sub-hierarchies, we can better deal with tasks that require degree of flexibility. For
```
Comment (Chenhui-Jia) [inline docs/about.rst:15]: we can better deal with tasks that require degree of flexibility ----> **a** degree of flexibility

**Review by Chenhui-Jia — APPROVED** [review]

File: docs/about.rst [hunk 2]
```diff
@@ -3,4 +3,16 @@ Why `repytah`
...
 Therefore, the goal of the Python package `repytah` is to extract these repetitions and their relationships 
-to each other in order to form aligned hierarchies.
\ No newline at end of file
+to each other in order to form aligned hierarchies, a low-dimensional representation for sequential data
+that synthesizes and aligns all possible hierarchies of the meaningful repetitive structure in a song along
+a common time axis. In addition to visualizing the structure of a song, aligned hierarchies can be
+embedded into a classification space with a natural notion of distance and post-processed to narrow the
+exploration of a music-based data stream to certain lengths of structure, or to address numerous MIR
+tasks, including the cover song task, the segmentation task, and the chorus detection task.
+
```
Comment (Chenhui-Jia) [inline docs/about.rst:12]: Should we mention that our package can go beyond the music area after the first paragraph? Something like this: Moreover, the python package `repytah` can go beyond the MIR area. People working with sequential data where repetitions have meaning can use `repytah` as a tool as well.

Comment (bwang64) [reply docs/about.rst:12]: Good idea!

File: docs/about.rst (hunk 1 above)
Comment (bwang64) [reply docs/about.rst]: Now changed!

**Review by thuntran — APPROVED** [review]
Looks great!!

**Comment by thuntran** [comment]
Hi @bwang64, can this be merged asap so that we can check the rendered version on readthedocs? Thank you!

**Pull request merged**